bitnami/grafana-operator/README.md

```diff
@@ -18,7 +18,7 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
 
 ## Prerequisites
 
-- Kubernetes 1.12+
+- Kubernetes 1.16+
```

Comment:
Sadly not because the `crds` directory is not templated.

Reply:
Not saying that you should, but it would be possible to move them to the `templates` directory, and then template them there. That would satisfy both the "old" and the "new" :)

Reply:
@tomberget I had a lot of issues with helm charts templating crds. It is still not mature. Also if the chart relies on the crds to be created beforehand.

Reply:
It also bricked a lot of our `helm upgrade` commands in our 100+ clusters when a chart decided to use templated crds.